storage/objects.go

```diff
@@ -205,6 +256,48 @@ func (s *ObjectsClient) Delete(ctx context.Context, input *DeleteObjectInput) er
 	return nil
 }
 
+// GetMpuInput represents parameters to a GetMpu operation
+type GetMpuInput struct {
+	PartsDirectoryPath string
+}
+
+type GetMpuHeaders struct {
```

Comment:
Why is the headers returned from `GetMpuOutput` limited to these two values?

Reply:
Those are the only values listed in the JSON response in the [documentation](https://github.com/joyent/manta-muskie/blob/master/docs/index.md#getmultipartupload-get-loginuploads0-fidstate). If there is documentation elsewhere that I can use to build a more complete list, that would be awesome.

Reply:
Here is an example request/response:
```
GET /elijah.zupancic/uploads/00b/00bf2f27-6370-c70b-bb29-b43784ae7b93/state HTTP/1.1
accept-version: ~1.0
Accept: application/json, */*
x-request-id: d578d434-0ef6-11e9-ae86-3748230df2b0
Date: Thu, 3 Jan 2019 01:28:11 GMT
Authorization: Signature keyId="/elijah.zupancic/keys/46:26:9a:c2:ef:0f:07:b2:ce:83:66:82:4f:3b:f7:f7",algorithm="rsa-sha256",signature="R3OAO2WP30iEsuf0C4Eo+6SbltGnO1eK35KNLQF0FOd99MHoVyHJzoclaLRCpU+XNVLCkJafWS0gKI2ltNMJuCxoaAfM+ZpzkRzAl1xxQXuiRExLBRlqsEGgtgVDnSl9YvLULbVcU5Ilg6zuF1q3bYcfEQzzGFYZXeNxKFBIk/kNneZ7spGx0rWW7jpxi5o4dyvRcO/YGyCJpYRWQ6xYq7/Ui9213tVBcv3LdxfEbsH6/pxQEHvUQQR6pA+oUEKgZfrl70XVNUHqmdVguZH1gXp10clKLtTXdZCjtZ8Vgy5k90ZjhMngG3fbHrV4f9g31MWlK90mvCjTHVWlSOZt7g=="
x-http-signing-time-ns: 245654
Host: us-east.manta.joyent.com
Connection: Keep-Alive
User-Agent: Java-Manta-SDK/3.3.1-SNAPSHOT (Java/1.8.0_162/Oracle Corporation)
Accept-Encoding: gzip,deflate

HTTP/1.1 200 OK
Content-Type: application/json
Content-Length: 677
Content-MD5: WGukq69WNC54VWn2lDPROw==
Date: Thu, 03 Jan 2019 01:27:32 GMT
Server: Manta
x-request-id: d578d434-0ef6-11e9-ae86-3748230df2b0
x-response-time: 452
x-server-name: 562a0e29-5024-4482-8f6c-26e0d95c5a36
Connection: Keep-alive

{"id":"00bf2f27-6370-c70b-bb29-b43784ae7b93","state":"done","targetObject":"/elijah.zupancic/stor/java-manta-integration-tests/ac7c6be5-b653-424c-a058-34bd28c65750/EncryptedServerSideMultipartManagerIT/2d811cf8-3c63-4590-9da8-e1c741925fc0","headers":{"m-encrypt-key-id":"integration-test-key","m-encrypt-iv":"F4nimOYmIM9ZKYxdh1IKEg==","m-encrypt-metadata":"","m-encrypt-metadata-hmac":"ruUylu7tJ6W/jys1B16MdA==","m-encrypt-type":"client/1","m-encrypt-cipher":"AES128/CTR/NoPadding","m-encrypt-metadata-iv":"RlaOEJkmTTBq4ql+h/DhBg==","m-encrypt-hmac-type":"HmacMD5"},"numCopies":2,"creationTimeMs":1543790652523,"partsMD5Summary":"eBs5bzKeZdZvWgrPfiPO8w==","result":"committed"}
```

With formatted json:
```json
{
	"id": "00bf2f27-6370-c70b-bb29-b43784ae7b93",
	"state": "done",
	"targetObject": "/elijah.zupancic/stor/java-manta-integration-tests/ac7c6be5-b653-424c-a058-34bd28c65750/EncryptedServerSideMultipartManagerIT/2d811cf8-3c63-4590-9da8-e1c741925fc0",
	"headers": {
		"m-encrypt-key-id": "integration-test-key",
		"m-encrypt-iv": "F4nimOYmIM9ZKYxdh1IKEg==",
		"m-encrypt-metadata": "",
		"m-encrypt-metadata-hmac": "ruUylu7tJ6W/jys1B16MdA==",
		"m-encrypt-type": "client/1",
		"m-encrypt-cipher": "AES128/CTR/NoPadding",
		"m-encrypt-metadata-iv": "RlaOEJkmTTBq4ql+h/DhBg==",
		"m-encrypt-hmac-type": "HmacMD5"
	},
	"numCopies": 2,
	"creationTimeMs": 1543790652523,
	"partsMD5Summary": "eBs5bzKeZdZvWgrPfiPO8w==",
	"result": "committed"
}
```

Reply:
Again, I think it's best to keep to the documented response values rather than any given response test. Once the documentation gets an upgrade we can come back around and update this. Otherwise I fear we could end up adding values that are not supported broadly or that may accidentally be changed by a later author of the Manta MPU code.